build.rs

```diff
@@ -1,11 +1,7 @@
-use std::str::FromStr;
-
-fn gdal_version_info() -> String {
-    gdal_sys::gdal_version_docs_rs_wrapper()
-}
+use std::{env, str::FromStr};
 
 fn main() {
-    let gdal_version_string = gdal_version_info();
+    let gdal_version_string = env::var("DEP_GDAL_VERSION_NUMBER").unwrap();
```

Comment:
Though very unlikely, consider using `.expect(...)` with a helpful message to indicate it was expected that `gdal-sys/build.rs` would have emitted `cargo:version-number`.

Reply:
[4e263c9](https://github.com/georust/gdal/pull/377/commits/4e263c94e97ca2cf1f2d468766125e9531178012)